packages/Button/src/Button.js

```diff
@@ -169,8 +177,16 @@ const Button = React.forwardRef((props, ref) => {
     kind,
   };
 
+  const hasForcedFocusAttribute = { "data-has-forced-focus": hasForcedFocus || null };
```

Comment:
Why the indirection? Can't this go right into line 188?

Reply:
because i dont want to pollute the dom with `data-has-forced-focus="false"`, which will be 99.9% of cases

Reply:
oh i understood that wrong... if it line breaks nicely, sure i can do that